webclient/impls/core/src/test/config/javascript/karma.conf.js

```diff
@@ -82,7 +82,7 @@ module.exports = function(config) {
     // the default configuration
     htmlReporter: {
       outputDir:    'target/test-reports-webclient/karma_html',
-      templatePath: 'node_modules/karma-html-reporter/jasmine_template.html'
+      templatePath: '../../../node_modules/karma-html-reporter/jasmine_template.html'
```

Comment:
Path was broken already...

Reply:
Reverting this. The path works when compilation is run from the root project's folder!

Reply:
Addressed in Addressed in https://github.com/webdetails/cde/pull/1047/commits/77b63e8d6f3f8534c882be53c14fc360ee3b0b55.